analyses/survival-analysis/survival-analysis_tp53_telomerase.Rmd

```diff
@@ -0,0 +1,407 @@
+>
+>_The regression coefficients._ The second feature to note in the Cox model results is the the sign of the regression coefficients (coef). A positive sign means that the hazard (risk of death) is higher, and thus the prognosis worse, for subjects with higher values of that variable. The variable sex is encoded as a numeric vector. 1: male, 2: female. The R summary for the Cox model gives the hazard ratio (HR) for the second group relative to the first group, that is, female versus male. The beta coefficient for sex = -0.53 indicates that females have lower risk of death (lower survival rates) than males, in these data.
+>
+>_Hazard ratios._ The exponentiated coefficients (exp(coef) = exp(-0.53) = 0.59), also known as hazard ratios, give the effect size of covariates. For example, being female (sex=2) reduces the hazard by a factor of 0.59, or 41%. Being female is associated with good prognostic.
+>
+>_Confidence intervals of the hazard ratios._ The summary output also gives upper and lower 95% confidence intervals for the hazard ratio (exp(coef)), lower 95% bound = 0.4237, upper 95% bound = 0.816.
+>
+>_Global statistical significance of the model._ Finally, the output gives p-values for three alternative tests for overall significance of the model: The likelihood-ratio test, Wald test, and score logrank statistics. These three methods are asymptotically equivalent. For large enough N, they will give similar results. For small N, they may differ somewhat. The Likelihood ratio test has better behavior for small sample sizes, so it is generally preferred.
+>
+> Below, HR = exp(coeff) 
+
+
+
+Note we can't use the `survival_analysis()` function here because of the multivariate nature of the terms; that function requires only ONE independent variable. Therefore we use a separate function for the multivariate models, defined in `util/survival_multivariate.R`. This function visualizes and prints the model results, and exports the fitted model as a TSV table. This function does not currently export the _plot_.
+
+
+#### TP53 score * Telomerase score * HGG group
+
+```{r multivar-fit-3terms}
+fit_plot_save_cox_multivariate(meta_indep_hgg, 
+                          "tp53_score*telomerase_score*hgg_group", 
+                          results_dir)
+```
+
+
+#### TP53 score * Telomerase score, for each cancer group.
+
+
+```{r multivar-fit-2terms}
+for (cg in cancer_groups_of_interest_df$cancer_group) {
+  cat("==============", cg, "==================")
+  # filter to cancer group of interest
+  fit_data <- cancer_groups_of_interest_df %>% 
+    filter(cancer_group == cg)
+  
+  # Define filename where model will be saved
+  filename <- paste0("cox_reg_results_per_tp53_score-by-telomerase_score_",
+                     stringr::str_replace_all(cg, " ","-"),
+                     ".tsv")
+  
+  # Fit model
+  fit_plot_save_cox_multivariate(meta_indep, 
```

Comment:
Pretty sure this is the source of the bug @jharenza mentioned

```suggestion
  fit_plot_save_cox_multivariate(fit_data, 
```

Reply:
updated this, but also was a bug above, too